spotbugs/etc/messages.xml

```diff
@@ -9128,6 +9137,51 @@ Using floating-point variables should not be used as loop counters, as they are
       ]]>
         </Details>
   </BugPattern>
+
+  <BugPattern type="ENV_USE_PROPERTY_INSTEAD_OF_ENV">
+    <ShortDescription>It is preferable to use portable Java property instead of environment variable.</ShortDescription>
+    <LongDescription>It is preferable to use portable Java property '{3}' instead of environment variable '{2}' in method {1}.</LongDescription>
+    <Details>
+      <![CDATA[<p>
+        Environment variables are not portable, the variable itself (not only the value) may be different depending on the running OS.
```

Comment:
Minor nitpicking that "the variable **name** iftself" might be better here, but I'm French so please feel to disregard if you disagree with my broken English :)

Reply:
Thanks, this way it's more clear and unambiguous. English is also not my mother tongue 😄